package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -449,19 +452,46 @@ def write(self, selection, frame=None):
 class DumpReader(base.ReaderBase):
     """Reads the default `LAMMPS dump format`_
 
-    Expects trajectories produced by the default 'atom' style dump.
+    Supports coordinates in the LAMMPS "unscaled" (x,y,z), "scaled" (xs,ys,zs),
+    "unwrapped" (xu,yu,zu) and "scaled_unwrapped" (xsu,ysu,zsu) coordinate
+    conventions (see https://docs.lammps.org/dump.html for more details).
+    If no coordinate convention is provided, or if "auto" is selected,
```

Comment:
Then why have both, why not just have `None` guess the order?

Reply:
I don't think the code actually supports `None`, by the way -- it checks if the variable is in a list of strings. Perhaps

```suggestion
    If `lammps_coordinate_convention='auto'` (default),
```